src/bootstrap/config/tests.rs

```diff
@@ -94,3 +92,24 @@ fn detect_src_and_out() {
 fn clap_verify() {
     Flags::command().debug_assert();
 }
+
+#[test]
+fn override_toml() {
+    let config = Config::parse_inner(
+        &[
+            "check".to_owned(),
+            "--config=/does/not/exist".to_owned(),
+            "--options=rust.lto=fat".to_owned(),
```

Comment:
I would also test overriding multiple values.

--

The PR looks good to me. Do you want to make a review as well? @jyn514

Reply:
Found a bug in merging with testing multiple values, good call 😄